Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -500,46 +444,72 @@ - (void)videoAdStarted:(NSDictionary *)properties withOptions:(NSDictionary *)in
     // StreamingAnalytics's asset. This is because ns_st_ci will have already been set via asset_id in a
     // Content Started calls (if this is a mid or post-roll), or via content_asset_id on Video Playback
     // Started (if this is a pre-roll).
-    NSString *contentId = [[[self.streamAnalytics playbackSession] asset] label:@"ns_st_ci"] ?: @"0";
+    
+    NSDictionary *labels = self.streamAnalytics.configuration.labels;
+    NSString *contentId = [labels objectForKey:@"ns_st_ci"] ?: @"0";
+   
     NSMutableDictionary *mapWithContentId = [NSMutableDictionary dictionaryWithDictionary:map];
     [mapWithContentId setValue:contentId forKey:@"ns_st_ci"];
+    
+    [self addMetaDataAssets:mapWithContentId withOptions:integrations];
 
-    [[self.streamAnalytics playbackSession] setAssetWithLabels:mapWithContentId];
-    SEGLog(@"[[SCORStreamingAnalytics playbackSession] setAssetWithLabels:%@", mapWithContentId);
+    [self movePosition: properties];
```

Comment:
Since this is considered a start event do we need the position here?

Reply:
Yes, according to [page28](https://app.box.com/s/csd2t9sb9z0m6gdadg852ysl4724jac8/file/545661486639), we need to call it.

Reply:
Yes, it needs to be called according to documentation on page 28 as well.